### PR TITLE
Hw22 jpa

### PR DESCRIPTION
**Pull request by u2107680007911543774e7r**

### CODE REVIEW

**Review by Artsapollo — APPROVED** [review]
Good

File: src/main/java/homework_jpa/CustomerDao.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package homework_jpa;
+
+import java.math.BigDecimal;
+import java.sql.SQLException;
+
+public interface CustomerDao {
+	boolean insertCustomer(Customer customer) throws SQLException;
```
Comment (SerhiiPasko) [inline src/main/java/homework_jpa/CustomerDao.java]: Why you throws SQLException?

File: src/test/java/lesson_jpa/OrderDaoImplTest.java [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
 import java.sql.SQLException;
-import java.util.Set;
 
 import org.junit.FixMethodOrder;
 import org.junit.Test;
 import org.junit.runners.MethodSorters;
 
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class OrderDaoImplTest {
-	private static final BigDecimal NOT_EXIST_ORDER = BigDecimal.valueOf(1111110);
```
Comment (SerhiiPasko) [inline src/test/java/lesson_jpa/OrderDaoImplTest.java:19]: It is not good. You crash your jdbc examples =(

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Goot, but you must delete all don't need throws

File: src/test/java/lesson_jpa/OrderDaoImplTest.java (hunk 2 above)
Comment (u2107680007911543774e7r) [reply src/test/java/lesson_jpa/OrderDaoImplTest.java:19]: No, I don't crush JDBC, it's just 2 commit for jpa hw...

**Pull request merged**